README.md

```diff
@@ -4,19 +4,17 @@ Kód je uvolněn pod [licencí MIT](https://opensource.org/licenses/MIT). Na tex
 
 ## Technické řešení
 
-Web je statický, hostovaný na [Netlify](https://www.netlify.com) a generovaný pomocí [Jekylla](http://jekyllrb.com). Zde je uložené pouze HTML a související data, zejména tedy nikoliv fotky a větší data.
+Web je statický, hostovaný na [Vercelu](https://vercel.com) a generovaný pomocí [Jekylla](http://jekyllrb.com). Zde je uložené pouze HTML a související data, zejména tedy nikoliv fotky a větší data.
```

Comment:
https://www.vercel.com?utm_source=[ohlasy]&utm_campaign=oss

**Correct link for Vercel sponsorship**

Reply:
Thank you, fixed!